distribution/ddf-common/src/main/resources/bin/setenv-wrapper.conf

```diff
@@ -41,5 +41,5 @@ wrapper.java.additional.26 = -XX:OnError=%KARAF_HOME%/bin/ddf_on_error.sh wrappe
 
 # Set the JVM initial and max heap space as desired
 wrapper.java.initmemory = 2048
-wrapper.java.maxmemory = 4096
+wrapper.java.maxmemory = 6144
```

Comment:
this won't be so simple because 4gb is the max value allowed in wrapper.java.maxmemory https://cwiki.apache.org/confluence/display/ACTIVEMQ/Java+Service+Wrapper you can add `wrapper.java.additional.25=-Xmx8g` but it will still be overridden by the default wrapper.java.maxmemory=512 which I think has to be manually removed after

Reply:
Yeah I'm not sure I like that. It sounds like we need to come up with a different option for the service wrapper.

Reply:
👍 sgtm